lib/pfconfig/util.pm

```diff
@@ -17,13 +17,35 @@ Utilities function for pfconfig
 use strict;
 use warnings;
 
+=head2 config_file_path
+
+Returns the config file path
+
+=cut
+
+sub config_file_path {
+    return "/usr/local/pf/conf/pfconfig.conf";
+}
+
+=head2 control_file_path
+
+Returns the control file path for a namespace
+
+=cut
+
 sub control_file_path {
     my ($namespace) = @_;
-    return "/usr/local/pf/var/control/".$namespace."-control";
+    return "/usr/local/pf/var/control/" . $namespace . "-control";
 }
 
+=head2 socket_path
+
+Returns the path of the socket on the system
+
+=cut
+
 sub socket_path {
-    return '/usr/local/pf/var/run/pfconfig.sock';  
+    return '/usr/local/pf/var/run/pfconfig.sock';
 }
```

Comment:
I taught I'd keep the possibility of having it dynamic in case we decide to have multiple sockets but I think that will be useless. I'll remove it